db/db_impl/db_impl.h

```diff
@@ -1871,9 +1867,9 @@ class DBImpl : public DB {
   Status DelayWrite(uint64_t num_bytes, WriteThread& write_thread,
                     const WriteOptions& write_options);
 
-  // Begin stalling of writes when memory usage increases beyond a certain
-  // threshold.
-  void WriteBufferManagerStallWrites();
+  // If stall conditions are met, begin stalling of writes with help of
+  // `WriteBufferManager`
+  void MaybeWriteBufferManagerStallWrites();
```

Comment:
Cleanup 7: renaming this function to reflect that we will still do stall condition check again within this function.